modules/yahoosspBidAdapter.js

```diff
@@ -184,46 +184,46 @@ function validateAppendObject(validationType, allowedKeys, inputObject, appendTo
       case 'string':
         if (allowedKeys.indexOf(objectKey) !== -1 && isStr(inputObject[objectKey])) {
           outputObject[objectKey] = inputObject[objectKey];
-        };
+        }
         break;
       case 'number':
         if (allowedKeys.indexOf(objectKey) !== -1 && isNumber(inputObject[objectKey])) {
           outputObject[objectKey] = inputObject[objectKey];
-        };
+        }
         break;
 
       case 'array':
         if (allowedKeys.indexOf(objectKey) !== -1 && isArray(inputObject[objectKey])) {
           outputObject[objectKey] = inputObject[objectKey];
-        };
+        }
         break;
       case 'object':
         if (allowedKeys.indexOf(objectKey) !== -1 && isPlainObject(inputObject[objectKey])) {
           outputObject[objectKey] = inputObject[objectKey];
-        };
+        }
```

Comment:
please keep all these semicolons, we plan to turn on a no implied semicolon linting rule in the future

Reply:
addressed